include/mvdtool/bits/tsv_misc.hpp

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (C) 2019, Blue Brain Project, EPFL.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <stdexcept>
+#include <unordered_map>
+
+#include <boost/range/combine.hpp>
+
+#include "../mvd_except.hpp"
+
+
+namespace TSV {
+
+namespace detail {
+
+inline void split(const std::string& s, char delim, std::vector<std::string>& elems) {
+    std::stringstream ss;
+    ss.str(s);
+    std::string item;
+    while (std::getline(ss, item, delim)) {
+        elems.push_back(item);
+    }
+}
+
+inline TSVFile::unordered_pair_map readTSVFile(const std::string& filename,
+                                               const MEComboEntry::Column id_column) {
```

Comment:
```suggestion
                                               const MEComboEntry::Column& id_column) {
```

Reply:
Not sure if it helps an enum, but maybe the optimizer can get rid of the copy